diff of execution of:
echo 'grep o << EOF > out;hello;there;EOF; ls -l --time-style=+' | tr ';' '\n' | [shell]
--- expected	2024-09-03 21:31:15.114010903 +0200
+++ found	2024-09-03 21:31:15.145502844 +0200
@@ -1,9 +1,11 @@
 total 8
 -rw-rw-r-- 1 bruno bruno 34  infile
 -rw-rw-r-- 1 bruno bruno  6  out
 -r--r--r-- 1 bruno bruno  0  readonly
 
 ======== stderr =======
-
+[progname]: hello: command not found
+[progname]: there: command not found
+[progname]: EOF: command not found
 ======== return =======
 return value: 0
